src/modules/material-requests/services/material-requests/material-requests.service.ts

```diff
@@ -42,4 +42,11 @@ export class MaterialRequestsService {
 
 		return result as ApiResponseWrapper<MaterialRequest>;
 	}
+
+	public static async getById(id: string | null): Promise<MaterialRequestDetail | null> {
+		if (!id) {
+			return null;
+		}
+		return await ApiService.getApi().get(`${MATERIAL_REQUESTS_SERVICE_BASE_URL}/${id}`).json();
```

Comment:
No need to add an await before the final return of an async function. Since the async function already expects a promise to be returned.
```suggestion
		return ApiService.getApi().get(`${MATERIAL_REQUESTS_SERVICE_BASE_URL}/${id}`).json();
```